src/main.ml

```diff
@@ -61,9 +61,9 @@ let parseOneFile (fname: string) : C.file =
 
   if (not (Feature.enabled "epicenter")) then (
     (* sm: remove unused temps to cut down on gcc warnings  *)
```

Comment:
Is this comment actually up-to-date? Or does it just remove all unused code?

Reply:
Looks like it removes all unused code.

Reply:
Maybe we can also update the text here then?

Reply:
Once we update this text, I think it's good to merge!

Reply:
c2e97b91e380f8a19ea6343e2179e660f60c8526 removed the entire epicenter feature check, not just updated the comment.

Reply:
The epicenter feature has long been removed from our fork, and the upstream version is no longer compatible with the current code base without rewriting.